### PR TITLE
IP hidden by default

### PR DESCRIPTION
**Pull request by ignasbernotas**
kapture 2018-09-25 at 22 38 53

### CODE REVIEW

File: src/renderer/pages/vpn.vue [hunk 1]
```diff
@@ -23,8 +23,10 @@
           :class="{'is-grey':statusCode===-1}"
           v-text="statusTitle"/>
         <div
-          class="control__location"
-          v-if="ip">current IP: {{ ip }}</div>
+          class="control__location">
+          current IP:
+          <span :class="{'text-blurry': ip}">{{ ip ? ip : 'Refreshing..' }}</span>
```
Comment (donce) [inline src/renderer/pages/vpn.vue]: `ip || 'Refreshing...'` ?

File: src/renderer/store/modules/connection.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ class ActionLooperConfig {
 const defaultStatistics = {
 }
 
-const IP_REFRESHING = 'Refreshing...'
+const IP_UNAVAILABLE = null
```
Comment (donce) [inline src/renderer/store/modules/connection.js]: Maybe we can skip this constant?
If you want to keep this constant, we should reuse it in `const state = { ip: null, ... }`

File: src/renderer/store/modules/connection.js [hunk 3]
```diff
@@ -191,13 +191,13 @@ function actionsFactory (
...
       if (newStatus === ConnectionStatusEnum.CONNECTED) {
-        commit(type.CONNECTION_IP, IP_REFRESHING)
+        commit(type.CONNECTION_IP, IP_UNAVAILABLE)
         const statisticsLooperConfig =
           new ActionLooperConfig(type.CONNECTION_STATISTICS, config.statisticsUpdateThreshold)
         await dispatch(type.START_ACTION_LOOPING, statisticsLooperConfig)
       }
       if (newStatus === ConnectionStatusEnum.NOT_CONNECTED) {
-        commit(type.CONNECTION_IP, IP_REFRESHING)
+        commit(type.CONNECTION_IP, IP_UNAVAILABLE)
```
Comment (donce) [inline src/renderer/store/modules/connection.js]: !! This broke the tests - please update them.

File: src/renderer/pages/vpn.vue [hunk 4]
```diff
@@ -23,8 +23,10 @@
           :class="{'is-grey':statusCode===-1}"
           v-text="statusTitle"/>
         <div
-          class="control__location"
-          v-if="ip">current IP: {{ ip }}</div>
+          class="control__location">
+          current IP:
+          <span :class="{'text-blurry': ip}">{{ ip || 'Refreshing..' }}</span>
```
Comment (donce) [inline src/renderer/pages/vpn.vue]: Previously, we didn't show this label at all until first ip fetch passed - now we'll show 'current IP: Refreshing..'. This is probably intentional?

Comment (ignasbernotas) [reply src/renderer/pages/vpn.vue]: Yep:)

**Pull request merged**